app/cli/cmd/rewind.go

```
@@ -38,71 +58,445 @@ func rewind(cmd *cobra.Command, args []string) {
+}
+
+func formatLogMessage(msg string) string {
+	var res string
+
+	// Check for message type patterns
+	switch {
+	case strings.Contains(msg, "User prompt"):
+		res = "💬 " + msg
+	case strings.Contains(msg, "Gpt4cli reply"):
+		if coins := regexp.MustCompile(`(\d+) 🪙`).FindStringSubmatch(msg); len(coins) >= 2 {
+			res = "🤖 AI Response | " + coins[1] + " 🪙"
```

golangci-lint (1.64.8)

ineffectual assignment to res

(ineffassign)